app/jobs/save_company_with_missing_identifiers_job.rb

```diff
@@ -2,9 +2,11 @@ class SaveCompanyWithMissingIdentifiersJob < ApplicationJob
   queue_as :default
 
   def perform(submission)
+    company_name = submission.extra.dig('params', 'company_name')
```

Comment:
@jsuchal doteraz sme neukladali nazov spolocnosti (ktory budeme asi chciet poznat kvoli emailu), iba ICO. Dalo by sa to pri vytvarani emailu tahat z autoformu, ale radsej by som to tam asi vopred doplnila.
Pri novych by sme to zacali uz ukladat. pri tych doteraz vytvorenych by som to jednorazovo doplnila cez autoform. Alebo navrhujes inak?

Reply:
Ano toto asi dava zmysel, aj ked teda autoform limitu by som sa uplne nebal, lebo ten vieme aj navysit tusim.

Reply:
Pozerala som, asi iba celoplosne, nie je to per client.
Ak teda suhlasis s tymto postupom, za mna je toto ready.